modules/apps/chargen/skills-talents.js

```diff
@@ -208,6 +283,8 @@ export class SkillsTalentsStage extends ChargenStage {
       valid = false
     }
 
+    // Should this allow partial allocation? I think it should not validate if allocated value is anything other than 40
+    // simply because it could be done by accident and hard to retrace in such a case. /Forien
```

Comment:
@moo-man Is it intentional to allow submitting the stage if not all points were allocated?

Reply:
Same with skills, is it intentional to allow submitting stage if less than 3 skills have been selected for each advancement "tier"?

Reply:
I didn't have any specific reasons for validating this or not validating that, just what seemed important/easy to mess up.